lib/broccoli/fastboot-app-factory-plugin.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-env node */
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+const Plugin = require('broccoli-plugin');
+const fastbootAppFactoryModule = require('../utilities/fastboot-app-factory-module');
+
+module.exports = class FastBootAppFactory extends Plugin {
+  constructor(inputNode, options) {
+    super([inputNode], {
+      annotation: 'Generate: FastBoot app-factory module',
+      persistentOutput: true
+    });
+    this.appName = options.appName;
+    this.isModuleUnification = options.isModuleUnification;
+  }
+
+  build() {
+    let outputPath = path.join(this.outputPath, 'app-factory.js');
```

Comment:
this join is not needed. fs.readFileSync(`${this.outputPath}/app-factory.js`) will suffice.